[2025-10-18T18:44:09.006+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-18T18:44:09.027+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flood_prediction_pipeline.fetch_weather_data manual__2025-10-18T18:44:07.687317+00:00 [queued]>
[2025-10-18T18:44:09.035+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flood_prediction_pipeline.fetch_weather_data manual__2025-10-18T18:44:07.687317+00:00 [queued]>
[2025-10-18T18:44:09.036+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-18T18:44:09.048+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): fetch_weather_data> on 2025-10-18 18:44:07.687317+00:00
[2025-10-18T18:44:09.055+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=533) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-18T18:44:09.056+0000] {standard_task_runner.py:63} INFO - Started process 536 to run task
[2025-10-18T18:44:09.055+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'flood_prediction_pipeline', 'fetch_weather_data', 'manual__2025-10-18T18:44:07.687317+00:00', '--job-id', '121', '--raw', '--subdir', 'DAGS_FOLDER/flood_prediction_dag.py', '--cfg-path', '/tmp/tmplbrmkgu2']
[2025-10-18T18:44:09.057+0000] {standard_task_runner.py:91} INFO - Job 121: Subtask fetch_weather_data
[2025-10-18T18:44:09.100+0000] {task_command.py:426} INFO - Running <TaskInstance: flood_prediction_pipeline.fetch_weather_data manual__2025-10-18T18:44:07.687317+00:00 [running]> on host fcca755cbcb6
[2025-10-18T18:44:09.172+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='flood_prediction_pipeline' AIRFLOW_CTX_TASK_ID='fetch_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2025-10-18T18:44:07.687317+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-18T18:44:07.687317+00:00'
[2025-10-18T18:44:09.173+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-18T18:44:09.212+0000] {logging_mixin.py:188} INFO - âœ… Weather data fetched: {'temp': 27.484974216485263, 'humidity': 90.4267805112827, 'pressure': 1011.418364270507, 'wind_speed': 16.641833591575597, 'rain_1h': 17.30818469502576, 'rain_3h': 68.21090427302448, 'rain_6h': 142.01023833079958, 'rain_24h': 264.4239364242638, 'rain_intensity': 5.42051602754243, 'temp_delta': 4.9850413733466645, 'humidity_delta': 7.993457032938579, 'year': 2025, 'month': 10, 'day': 18, 'hour': 18, 'minute': 44, 'second': 9}
[2025-10-18T18:44:09.213+0000] {python.py:237} INFO - Done. Returned value was: {'temp': 27.484974216485263, 'humidity': 90.4267805112827, 'pressure': 1011.418364270507, 'wind_speed': 16.641833591575597, 'rain_1h': 17.30818469502576, 'rain_3h': 68.21090427302448, 'rain_6h': 142.01023833079958, 'rain_24h': 264.4239364242638, 'rain_intensity': 5.42051602754243, 'temp_delta': 4.9850413733466645, 'humidity_delta': 7.993457032938579, 'year': 2025, 'month': 10, 'day': 18, 'hour': 18, 'minute': 44, 'second': 9}
[2025-10-18T18:44:09.213+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-18T18:44:09.233+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=flood_prediction_pipeline, task_id=fetch_weather_data, run_id=manual__2025-10-18T18:44:07.687317+00:00, execution_date=20251018T184407, start_date=20251018T184409, end_date=20251018T184409
[2025-10-18T18:44:09.271+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-18T18:44:09.285+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-10-18T18:44:09.289+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
